sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeIdentityDocumentsFromUrl.java

```diff
@@ -30,12 +30,12 @@
     public static void main(final String[] args) throws IOException {
         // Instantiate a client that will be used to call the service.
         FormRecognizerClient client = new FormRecognizerClientBuilder()
-            .credential(new AzureKeyCredential("{key}"))
-            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .credential(new AzureKeyCredential("3e571e46580141aaae155f0099a5a54b"))
```

Comment:
is this a real key?

Reply:
I don't see you online right now so I rotated the key for `mariari-canada-central`. Please check if anything else needs to be done to properly scrub the secret.

Reply:
Sorry. Will follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/101/Leaked-secret-procedure